typings/rendition.d.ts

```diff
@@ -628,3 +628,36 @@ declare module 'rendition/dist/unstable' {
 
 	class Form extends RenderableElementWithProps<FormProps, any> {}
 }
+
+declare module 'rendition/dist/extra/Form/mermaid' {
+
+}
+
+declare module 'rendition/dist/extra/Mermaid' {
+	import { BoxProps, RenderableElementWithProps } from 'rendition';
+	interface MermaidProps extends BoxProps {
+		value: string;
+	}
+
+	class Form extends RenderableElementWithProps<MermaidProps, any> {}
+}
+
+declare module 'rendition/dist/extra/Mermaid' {
+	import { BoxProps, RenderableElementWithProps } from 'rendition';
+
+	interface MermaidProps extends BoxProps {
+		value: string;
+	}
+
+	class Mermaid extends RenderableElementWithProps<MermaidProps, any> {}
+}
+
+declare module 'rendition/dist/extra/Markdown' {
```

Comment:
Can we move this above the Mermaid module declaration so that the typings have the same order as our files?
Moreover we might at some point want to consider splitting this huge declaration file into smaller ones, at least for the different module declarations.